### PR TITLE
Grafana notification policy

### PR DESCRIPTION
**Pull request by mirii1994**
Add support to the Grafana Notification Policy API.

### CODE REVIEW

File: grafana_notification_policies/client_grafana_notification_policy_get.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		ApiAction:    operationGetGrafanaNotificationPolicy,
+		ResourceName: grafanaNotificationPolicyResourceName,
+	})
+
+	if err != nil {
+		return grafanaNotificationPolicy, err
+	}
+
+	err = json.Unmarshal(res, &grafanaNotificationPolicy)
+	if err != nil {
```
[the remark this answers is not in the document]
Comment (mirii1994) [reply grafana_notification_policies/client_grafana_notification_policy_get.go:36]: #cycode_sast_ignore_here

File: grafana_notification_policies/client_grafana_notification_policy.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+	Routes         []GrafanaNotificationPolicy `json:"routes,omitempty"`
+}
+
+type GrafanaNotificationPolicy struct {
+	Continue          bool                        `json:"continue"`
+	GroupBy           []string                    `json:"group_by,omitempty"`
+	GroupInterval     string                      `json:"group_interval,omitempty"`
+	GroupWait         string                      `json:"group_wait,omitempty"`
+	MuteTimeIntervals []string                    `json:"mute_time_intervals,omitempty"`
+	ObjectMatchers    MatchersObj                 `json:"object_matchers,omitempty"`
```
Comment (yotamloe) [inline grafana_notification_policies/client_grafana_notification_policy.go:42]: Is there a way to include multiple matchers? Take for example a case where you want to route the notifications based on values of 2 or more labels

Comment (mirii1994) [reply grafana_notification_policies/client_grafana_notification_policy.go:42]: Yes, the matcher object holds a list of strings

**Review by yotamloe — APPROVED** [review]
Nicely done!

**Pull request merged**